### PR TITLE
feat(euchre): grey out partner when going alone (#1921)

### LINKED ISSUE #1921: Euchre (ユーカー) における「ゴーアローン (Go Alone)」時のパートナー休止状態の視覚化
## 課題の概要 (Problem)
Euchre には「ゴーアローン（一人で勝負）」という宣言があり、これを行うと宣言者のパートナーはそのラウンド（5トリック）の間、プレイに参加せず休みとなります（`Euchre.go` の仕様）。
現状、スコアやメッセージで「ゴーアローン」が行われていることは表示されますが、トリックが進行する盤面（`TrickDisplay`）やプレイヤーアイコンにおいて、パートナーが「非アクティブ」であることが直感的に伝わりません。

### 影響箇所 (User Impact)
- Euchre Webページ (`frontend/src/pages/EuchrePage.tsx`)
- トリック・プレイヤー表示コンポーネント
- **ユーザーへの影響**: ラウンド中「なぜ3人しかカードを出さないのか」が一瞬分かりにくく、ゲームのコンテキスト（状況把握）を見失いやすい。

## 改善提案 (Proposed Solution)
ゴーアローン宣言によって休止状態（`goingAlonePlayerIdx` のパートナー）になったプレイヤースロットに対して、視覚的なミュート（グレーアウト）処理を施します。

1. **アバターとカード領域のグレーアウト**:
   該当プレイヤーのエリア全体に `opacity-40` やグレースケールフィルタを適用します。
2. **状態バッジのオーバーレイ**:
   休んでいるプレイヤーの領域に「💤 Sitting out（休み）」といったテキストまたはアイコンをオーバーレイ表示させ、システムとして一時的にゲームから除外されていることを明確に示します。

## 期待される効果 (Expected Impact)
- **重大度: LOW** — ゲームの進行自体は可能だが、トリックテイキングにおける「誰が参加しているか」という重要なルールのコンテキストをUIとして適切に伝えるための改善。

### PR DESCRIPTION
**Pull request by yuta-yoshinaga**
Closes #1921

## Summary
- New `euchreSittingOutIdx` util resolves the teammate of the
  going-alone player from `state.players`/`state.goingAlonePlayerIdx`.
- EuchrePage applies `opacity-40 grayscale` to that CPU's row and
  renders a 💤 `sittingOut` badge in both the inline desktop layout
  and the mobile `<details>` panel.
- Returns null safely when the going-alone player has no teammate
  on their team (edge case).

## Test plan
- [x] `bun run test -- --run euchreSittingOut` (4 passed)
- [x] `bun run test -- --run EuchrePage` (65 passed; 2 new cases)
- [x] `bun run check`
- [ ] CI 緑
- [ ] `/euchre` で「一人で勝負」を選んだとき、パートナーの行がグレーアウト＋💤バッジになること
- [ ] 通常ラウンドでは普通に表示されること

### CODE REVIEW

**Comment by yuta-yoshinaga** [comment]
@claude

File: frontend/src/pages/EuchrePage.tsx [hunk 1]
```diff
@@ -336,27 +338,51 @@ function EuchrePageContent() {
...
+                              <span data-testid={`eu-sitting-out-${p.id}`} className="ml-2 text-ds-warning">
+                                💤 {t('sittingOut')}
+                              </span>
+                            )}
+                          </div>
                         </div>
-                      </div>
-                    ))
+                      );
+                    })
```
[the remark this answers is not in the document]
Comment (yuta-yoshinaga) [reply frontend/src/pages/EuchrePage.tsx:371]: Done in 2e7eafac — extracted the shared i18n stanza (name / cards / team / tricks / dealer suffix / sitting-out badge) into a `CpuPlayerLine` component used by both layouts. Kept the outer wrappers inline so the structural difference between the mobile `<details>` and desktop card stays visible at the call site.
